routes/author/index.js

```diff
@@ -118,14 +127,14 @@ module.exports = function(req, res){
           prev: {
             pageId: authorPrev.pageId,
             name: authorPrev.name,
-            isNotExist: authorPrev.isNotExist
+            isNotExist: authorPrev.isNotExist,
```

Comment:
__[jshint]__
*  Expected an assignment or function call and instead saw an expression.

*  Missing semicolon.

*  Expected '}' to match '{' from line 126 and instead saw ':'.

*  Expected an assignment or function call and instead saw an expression.

*  Missing semicolon.

*  Unrecoverable syntax error. (89% scanned).